mne_bids/mne_bids.py

```diff
@@ -594,18 +565,6 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     data_meta_fname = make_bids_filename(
         subject=subject_id, session=session_id, task=task, run=run,
         acquisition=acquisition, suffix='%s.json' % kind, prefix=data_path)
-    if ext in ['.fif', '.ds', '.vhdr', '.edf', '.bdf', '.set', '.cnt']:
-        raw_file_bids = make_bids_filename(
-            subject=subject_id, session=session_id, task=task, run=run,
-            acquisition=acquisition, suffix='%s%s' % (kind, ext))
-    else:
-        raw_folder = make_bids_filename(
-            subject=subject_id, session=session_id, task=task, run=run,
-            acquisition=acquisition, suffix='%s' % kind)
-        raw_file_bids = make_bids_filename(
-            subject=subject_id, session=session_id, task=task, run=run,
-            acquisition=acquisition, suffix='%s%s' % (kind, ext),
-            prefix=raw_folder)
     events_tsv_fname = make_bids_filename(
```

Comment:
`events_tsv_fname` should be `events_fname` ... or else all other `XXX_fname` should be `XXX_tsv_fname`/`XXX_json_fname`

Reply:
same comment as above. Let's make one PR do one thing ...

Reply:
okay done anyway :)